core/test/servico/LoginServiceTest.java

```diff
@@ -63,9 +63,9 @@ public void tearDown() {
     public void testeValidaUsuarioeSenha(){
 
         
-        Assert.assertTrue(LgnService.VerificarUsuarioESenha("Calebe", "123456"));
-        Assert.assertFalse(LgnService.VerificarUsuarioESenha("Calebe", ""));
-        Assert.assertFalse(LgnService.VerificarUsuarioESenha("noEcsiste", ""));
+        Assert.assertTrue(LgnService.verificarUsuarioESenha("Calebe", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 81).](https://www.codacy.com/app/calebepb/substituicao/file/2734604405/issues/source?bid=3308597&fileBranchId=3351368#l66)